src/EnergyPlus/HVACControllers.cc

```diff
@@ -2530,6 +2530,43 @@ void TraceIndividualController(EnergyPlusData &state,
     TraceFile.flush();
 }
 
+Real64 GetCurrentHVACTime(EnergyPlusData &state)
```

Comment:
📌 C++ a `time` library that parses and formats time strings and has other functions too. At some point, should probably just start using that as opposed to rolling our own implementations of all of that stuff.

Reply:
Yes, we should move a bunch of our stuff over, for sure.  That will be a great step forward.

Reply:
Except for that little detail of hour 24 which is both revered and reviled. https://github.com/NREL/EnergyPlus/issues/5360

Reply:
Revered by whom? Name one person. Other than yourself.

Reply:
Hah! I'm sure there's someone out there. According to [this page](https://www.timeanddate.com/time/am-and-pm.html), we can have both. "0:00 refers to midnight at the beginning of the day while 24:00 is midnight at the end of the day."

Reply:
There's a Schrodinger's Cat joke here somewhere.